config.json

```diff
@@ -855,6 +855,14 @@
       ]
     },
     {
+      "uuid": "1818f134-0ed9-e680-9b29-45ffd2b3344b0f841c7",
+      "slug": "markdown",
+      "difficulty": 3,
+      "topics": [
+        "refactoring"
```

Comment:
Not sure that this topic is appropriate, could you please explain your reasoning?

You can use this list of general topics common among all tracks - https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt

Reply:
`regular_expressions`, `pattern_matching` are good candidates for example

Reply:
Look at [this file](https://github.com/exercism/problem-specifications/blob/master/exercises/markdown/description.md):
"The markdown exercise is a refactoring exercise. There is code that parses a given string with Markdown syntax and returns the associated HTML for that string. Even though this code is confusingly written and hard to follow, somehow it works and all the tests are passing! Your challenge is to re-write this code to make it easier to read and maintain while still making sure that all the tests keep passing."

Reply:
@m-a-ge, this exercise is intended to be a refactoring exercise rather than an implementation one (see the README). This also explains the presence of a working solution (does TravisCI currently test this solution? I feel like it needs to) in `markdown.py`.

Reply:
@PaulGlass sorry, I've missed it due to amount of opened PRs 😕 
@N-Parsons thanks!

No changes needed then 😄